nav2_util/include/nav2_util/node_utils.hpp

```diff
@@ -48,11 +48,7 @@ std::string generate_internal_node_name(const std::string & prefix = "");
 
 /// Creates a node with a name as generated by generate_internal_node_name
 /**
- * Since this is an internal node, the node is created with the following
- * options.
- * options.use_global_arguments(false);
- * options.start_parameter_services(false);
- * options.start_parameter_event_publisher(false);
+ *  Creates a node with a name as generated by generate_internal_node_name.
```

Comment:
I think this comment is redundant with the line above it?

Reply:
fixed.